llvm/tools/llvm-stress/llvm-stress.cpp

```diff
@@ -49,6 +52,10 @@ static cl::opt<std::string> OutputFilename("o",
                                            cl::value_desc("filename"),
                                            cl::cat(StressCategory));
 
+static cl::opt<unsigned> MaxSizeCL(
+    "max-size", cl::desc("Max size of mutated module (defaults to 1 MiB)"),
+    cl::init(1048576), cl::value_desc("bytes"), cl::cat(StressCategory));
+
```

Comment:
Can you update these changes in `llvm/docs/CommandGuide/llvm-stress.rst` as well?